### PR TITLE
[WIP] jenkins_build.sh: Add support for multicontainer resin.io staging ins…

### PR DESCRIPTION
**Pull request by agherzan**
…tance

Signed-off-by: Andrei Gherzan <andrei@resin.io>

### CODE REVIEW

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -334,7 +341,7 @@ deploy_to_s3() {
 			su deployer<<EOSU
 echo "${BUILD_VERSION}" > "/host/images/${SLUG}/latest"
 /usr/src/app/node_modules/.bin/coffee /usr/src/app/scripts/prepare.coffee
-if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ]; then
+if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ] || [ "$deployto" = "multistaging" ]; then
```
Comment (floion) [inline automation/jenkins_build.sh]: Is the check where to deploy really necessary? We should not depend on that. The code check the appropriate S3 bucket to see if the image has been deployed there and acts accordingly
And the variable is wrong anyway, should actually be "deployTo" and not "deployto" so this code does not do the intended thing

File: automation/jenkins_build.sh [hunk 2]
```diff
@@ -334,7 +334,8 @@ deploy_to_s3() {
 		-e DEPLOYER_GID=$(id -g) \
 		-v $_s3_deploy_dir:/host/images resin/resin-img:master /bin/sh -x -e -c ' \
 			apt-get -y update
-			apt-get install -y s3cmd
+			apt-get install -y python-pip
+			pip install s3cmd
```
Comment (floion) [inline automation/jenkins_build.sh]: Why not on the same line? It's not too long

File: automation/jenkins_build.sh [hunk 3]
```diff
@@ -334,7 +342,7 @@ deploy_to_s3() {
 			su deployer<<EOSU
 echo "${BUILD_VERSION}" > "/host/images/${SLUG}/latest"
 /usr/src/app/node_modules/.bin/coffee /usr/src/app/scripts/prepare.coffee
-if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ]; then
+if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ] || [ "$deployTo" = "multistaging" ]; then
```
Comment (floion) [inline automation/jenkins_build.sh]: Is the check where to deploy really necessary? We should not depend on that. The code check the appropriate S3 bucket to see if the image has been deployed there and acts accordingly

**Comment by agherzan** [comment]
Rebased on current HEAD.

**Comment by agherzan** [comment]
@shaunmulligan @CameronDiver @pcarranzav Now that multi is in production, do we still need to support the multistaging instance? Feels to me that we can drop this PR.

**Comment by shaunmulligan** [comment]
@agherzan yeah we can close this

**Pull request closed without merging**